cmake/modules/FindBoost.cmake

```diff
@@ -2134,7 +2134,7 @@ if(Boost_FOUND)
         endif()
         if(_Boost_${UPPERCOMPONENT}_COMPILER_FEATURES)
           set_target_properties(Boost::${COMPONENT} PROPERTIES
-            INTERFACE_COMPILE_FEATURES "${_Boost_${UPPERCOMPONENT}_COMPILER_FEATURES}")
+            CXX_STANDARD 17)
```

Comment:
are we missing anything interesting from omitting the component _COMPILER_FEATURES here? or are those features just used as input to determine the value of CXX_STANDARD?